### PR TITLE
Add tests for ohapi/utils_fs.py

### LINKED ISSUE #14: Add tests
So far none of our functions have tests that could be run by a continuous integration. Having these would be great for making all of our lives easier. 

The OSFclient has some examples of how to test API code in Python that could be an inspiration: https://github.com/osfclient/osfclient/tree/master/osfclient/tests

### PR DESCRIPTION
**Pull request by kritisingh1**
Add tests for the functions defined in `ohapi/utils_fs.py`
## General Checkups
- [x] Have you checked that there aren't other open pull requests for the same issue/update/change?
- [x] If your code includes new features and not just bug fixes/copy editing: Did you include new tests?

## Related Issue

Reference #14

### CODE REVIEW

**Comment by kritisingh1** [comment]
@mcescalante I had initially went for adding individual names only but due to some unknown reason, there was an import error reported `could not find name strip_zip_suffix` and so I proceeded to importing * which fixed that error. Nevertheless I will try to look into it once again. Thanks for reviewing :)

**Comment by mcescalante** [comment]
This import line gets rid of linting errors (and passes tests) for me:

```
from ohapi.utils_fs import (strip_zip_suffix, guess_tags,
                            load_metadata_csv, validate_metadata)
```

File: ohapi/tests/test_test.py [hunk 1]
```diff
@@ -29,3 +31,14 @@ def test_load_metadata_csv(self):
         metadata_files = load_metadata_csv('ohapi/tests/data/'
                                            'metadata_proj_file_key_works.csv')
         self.assertEqual(len(metadata_files.keys()), 2)
+
+    def test_strip_zip_suffix(self):
```
Comment (gedankenstuecke) [inline ohapi/tests/test_test.py]: I think this test is already implicitly integrated by https://github.com/OpenHumans/open-humans-api/blob/master/ohapi/tests/test_test.py#L14. It checks that `.gz` and `.bz2` suffixes are stripped and that the corresponding uncompressed file types are still recognized. (the test coverage evaluation on code climate seems to agree that these lines are already covered). 

Do you think it's necessary to keep this test around?

Comment (kritisingh1) [reply ohapi/tests/test_test.py]: Yeah we can remove that. Thanks for reviewing. I'll update it soon along with adding more tests. :)

**Comment by gedankenstuecke** [comment]
I just wanted to bump this here, are you on this @kritisingh1 or would you need some help? 😃

**Comment by kritisingh1** [comment]
@gedankenstuecke I am so sorry for delay. I will be updating this as soon as possible. Thank you :slightly_smiling_face:

**Comment by madprime** [comment]
Merging now. Thanks! :)

**Pull request merged**